Prog/NCHousing_needs_units_targets.sas

```diff
@@ -0,0 +1,927 @@
+		else if hud_inc = 5 then max_rent=HHINCOME_a/12*costratio; *allow 120-200% above average to spend more; 
+	  if costratio <=.12 and hud_inc = 6 then max_rent=HHINCOME_a/12*.12; *avg for all HH hud_inc=6; 
+	  	else if hud_inc=6 then max_rent=HHINCOME_a/12*costratio; *allow 200%+ above average to spend more; 
+     
+	 *create flag for household could "afford" to pay more; 
+		couldpaymore=.;
+
+		if max_rent ~= . then do; 
+			if max_rent > rentgrs_a*1.1 then couldpaymore=1; 
+			else if max_rent <= rentgrs_a*1.1 then couldpaymore=0; 
+		end; 
+
+	
+    	*rent cost categories that make more sense for rents - no longer used in targets;
+		/*need to discuss for NC*/
+			rentlevel=.;
+			if 0 <=rentgrs_a<750 then rentlevel=1;
+			if 750 <=rentgrs_a<1200 then rentlevel=2;
+			if 1200 <=rentgrs_a<1500 then rentlevel=3;
+			if 1500 <=rentgrs_a<2000 then rentlevel=4;
+			if 2000 <=rentgrs_a<2500 then rentlevel=5;
+			if rentgrs_a >= 2500 then rentlevel=6;
+
+			mrentlevel=.;
+			if max_rent<750 then mrentlevel=1;
+			if 750 <=max_rent<1200 then mrentlevel=2;
+			if 1200 <=max_rent<1500 then mrentlevel=3;
+			if 1500 <=max_rent<2000 then mrentlevel=4;
+			if 2000 <=max_rent<2500 then mrentlevel=5;
+			if max_rent >= 2500 then mrentlevel=6;
+
+		 *rent cost categories now used in targets that provide a set of categories useable for renters and owners combined; 
+			/*need to discuss for NC*/
+			allcostlevel=.;
+			if rentgrs_a<800 then allcostlevel=1;
+			if 800 <=rentgrs_a<1300 then allcostlevel=2;
+			if 1300 <=rentgrs_a<1800 then allcostlevel=3;
+			if 1800 <=rentgrs_a<2500 then allcostlevel=4;
+			if 2500 <=rentgrs_a<3500 then allcostlevel=5;
+			if rentgrs_a >= 3500 then allcostlevel=6; 
+
+			mallcostlevel=.;
+
+			*for desired cost for current housing needs is current payment if not cost-burdened
+			or income-based payment if cost-burdened;
+
+			if costburden=1 then do; 
+
+				if max_rent<800 then mallcostlevel=1;
+				if 800 <=max_rent<1300 then mallcostlevel=2;
+				if 1300 <=max_rent<1800 then mallcostlevel=3;
+				if 1800 <=max_rent<2500 then mallcostlevel=4;
+				if 2500 <=max_rent<3500 then mallcostlevel=5;
+				if max_rent >= 3500 then mallcostlevel=6;
+
+			end; 
+
+			else if costburden=0 then do;
+
+				if rentgrs_a<800 then mallcostlevel=1;
+				if 800 <=rentgrs_a<1300 then mallcostlevel=2;
+				if 1300 <=rentgrs_a<1800 then mallcostlevel=3;
+				if 1800 <=rentgrs_a<2500 then mallcostlevel=4;
+				if 2500 <=rentgrs_a<3500 then mallcostlevel=5;
+				if rentgrs_a >= 3500 then mallcostlevel=6;
+
+			end; 
+
+
+	end;
+
+	
+	  		
+		
+  	else if ownershpd in ( 12,13 ) then do;
+
+	    ****** Owner units ******;
+	    
+	    Tenure = 2;
+
+		*create maximum desired or affordable owner costs based on HUD_Inc categories*; 
+
+		/*need to discuss for NC*/
+		if hud_inc in(1 2 3) then max_ocost=HHINCOME_a/12*.3; *under 80% of AMI then pay 30% threshold; 
```

Comment:
see above comment in renter section.

Reply:
The new costlevel cutoffs are 350, 700, 1050, 1500, 2500.

Reply:
@maggiesu0725 okay - you need to update the formats up top.